notifications_utils/recipients.py

```diff
@@ -63,12 +66,11 @@ def __init__(
         whitelist=None,
         template=None,
         remaining_messages=sys.maxsize,
-        sms_character_limit=0,
         international_sms=False,
     ):
         self.file_data = file_data.strip(', \n\r\t')
         self.template_type = template_type
-        self.character_limit = int(sms_character_limit) if self.template_type == 'sms' else sys.maxsize
+        self.character_limit = SMS_CHAR_COUNT_LIMIT if self.template_type == 'sms' else sys.maxsize
```

Comment:
We still need this `459` value somewhere - it's used in the error messages on both admin and api.

But happy for it to be elsewhere in utils. thoughts? Just in the __init__? `from notifications_utils import SMS_CHAR_COUNT_LIMIT`?